### DISCUSSION TITLE
Moya 15.1.0

### DISCUSSION THREAD

**Discussion by gre4ixin** [Moya/Moya#2260] [General]
Is it time for release **AsyncMoya**?)
@BasThomas

**Comment by cozzin** [comment]
I really want `AsyncMoya` to be released 🙏🙏

**Reply by gre4ixin** [reply]
waiting @BasThomas 🫣

**Reply by frankois941** [reply]
Still waiting, 
But I think adding a simple async 'request' method should be a good start

**Reply by gre4ixin** [reply]
> Still waiting, 
> 
> But I think adding a simple async 'request' method should be a good start

@frankois941 development branch already has an async/await implementation

**Reply by ImaginativeShohag** [reply]
When the next release will come?
